### PR TITLE
Update Windows cloudbuild config

### PR DESCRIPTION
**Pull request by pjh**
This sample needs to pull the builder from a different artifact registry location. 

Also, follows up on https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/222: we do not want to hard-code the region name or location in this file, the user should fill them in.

This PR aligns with https://cloud.google.com/kubernetes-engine/docs/tutorials/building-windows-multi-arch-images (and the changes I'm making to it in internal change 425411666).

### CODE REVIEW

**Comment by pjh** [comment]
@ibabou please review, thanks!

**Review by ibabou — APPROVED** [review]

**Comment by pjh** [comment]
Edited the title and the PR description to reflect that we also need to change where we pull the builder from.

**Comment by pjh** [comment]
ibabou@ could you lgtm again? Thanks!

**Review by ckim328 — APPROVED** [review]

**Comment by Shabirmean** [comment]
@ibabou ibabou is this good to be merged? Seems like your LGTM got refreshed, can you confirm?

**Comment by pjh** [comment]
This is ready to merge and I have the approvals to do so myself now. I'm waiting on review and submit of my internal docs CL before merging this.

**Comment by Shabirmean** [comment]
Thank you @pjh!

*Note:* to @GoogleCloudPlatform/anthos-dpe - please see comment https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/271#issuecomment-1027520101

**Pull request merged**
